src/h5web/visualizations/shared/models.ts

```diff
@@ -1,8 +1,28 @@
 import { ScaleLinear, ScaleSymLog, scaleSymlog, scaleLinear } from 'd3-scale';
 
+export type Size = { width: number; height: number };
+
 export type Domain = [number, number];
 
-export type Size = { width: number; height: number };
+export interface IndexAxisConfig {
+  indexDomain: Domain;
+  showGrid?: boolean;
+  isLog?: never; // invalid
```

Comment:
Because of how TypeScript works, `IndexAxisConfig | DataAxisConfig` doesn't prevent `isLog` from being declared in an index axis config. This line forces TypeScript to error, which helps users declare the config correctly.

If we add more properties to one config interface, we'll have to add it in this way to the other interface.